com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -48,7 +49,11 @@ public enum ScriptableRenderPassInput
         /// Executes a <c>ScriptableRenderPass</c> before rendering prepasses, f.ex, depth prepass.
         /// Camera matrices and stereo rendering are already setup at this point.
         /// </summary>
-        BeforeRenderingPrepasses = 150,
+        BeforeRenderingPrePasses = 150,
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("Obsolete, to match the capital from 'Prepass' to 'PrePass'")]
+        BeforeRenderingPrepasses = 151,
```

Comment:
Giving a second look at this, we can make this better by using ` (UnityUpgradable) -> ` defined here: https://confluence.unity3d.com/display/DEV/Obsolete+API+updaters

 So we avoid adding a new event and just rename the old one.

Reply:
Applied it :)